src/main/java/nl/tudelft/semgroup4/resources/Fonts.java

```diff
@@ -0,0 +1,15 @@
+package nl.tudelft.semgroup4.resources;
+
+import java.awt.Font;
+
+import org.newdawn.slick.TrueTypeFont;
+
+public abstract class Fonts {
```

Comment:
Why is this class abstract? Did you maybe mean final or static (i.e. cannot be instanced)?

< interface